### PR TITLE
optimization auto-build

### PR DESCRIPTION
**Pull request by Stevent-fei**

### CODE REVIEW

File: .github/workflows/auto-build.yml [hunk 1]
```diff
@@ -30,8 +30,10 @@ jobs:
           sudo tar -xvf sealer.tar.gz -C /usr/bin
           sudo sealer login ${{secrets.PRIVATEWAREHOUSE}}
           sudo sed -i "s/v1.20.14/$version/g" ./rootfs/etc/kubeadm.yml
+          sudo sed -i "s/v1.20.14/$version/g" ./rootfs/Metadata
+          sudo sed -i "s/amd64/$arch/g" ./rootfs/Metadata
+          sudo ./kubernetes/server/bin/kubeadm config images list --config ./rootfs/etc/kubeadm.yml 2>&1>>./rootfs/imageList
```
Comment (bxy4543) [inline .github/workflows/auto-build.yml:35]: If kubeadm is ARM architecture, it will fail.

File: .github/workflows/auto-build.yml [hunk 2]
```diff
@@ -30,8 +30,10 @@ jobs:
           sudo tar -xvf sealer.tar.gz -C /usr/bin
           sudo sealer login ${{secrets.PRIVATEWAREHOUSE}}
           sudo sed -i "s/v1.20.14/$version/g" ./rootfs/etc/kubeadm.yml
+          sudo sed -i "s/v1.20.14/$version/g" ./rootfs/Metadata
```
Comment (fanux) [inline .github/workflows/auto-build.yml:33]: @kakaZhou719 metadata already has version feild?

File: .github/workflows/auto-build.yml (hunk 1 above)
Comment (bxy4543) [inline .github/workflows/auto-build.yml:35]: change `2>&1`  to `2>/dev/null`

File: .github/workflows/auto-build.yml (hunk 2 above)
Comment (kakaZhou719) [reply .github/workflows/auto-build.yml:33]: no, here we need to write the value of "version" field.  @Stevent-fei .i mean we write a json file with content like `{
  "version": "v1.19.8",
  "arch": "amd64"
}
`

File: .github/workflows/auto-build.yml [hunk 3]
```diff
@@ -26,12 +26,16 @@ jobs:
...
           wget http://sealer.oss-cn-beijing.aliyuncs.com/auto-build/sealer.tar.gz
           sudo tar -xvf sealer.tar.gz -C /usr/bin
           sudo sealer login ${{secrets.PRIVATEWAREHOUSE}}
           sudo sed -i "s/v1.20.14/$version/g" ./rootfs/etc/kubeadm.yml
+          sudo sed -i "s/v1.20.14/$version/g" ./rootfs/Metadata
+          sudo sed -i "s/amd64/$arch/g" ./rootfs/Metadata
+          sudo ./kubernetes/server/bin/kubeadm config images list --config ./rootfs/etc/kubeadm.yml 2>/dev/null>>./rootfs/imageList
           cd ./rootfs
-          sudo ./bin/kubeadm config images list --config ./etc/kubeadm.yml 2>&1>>imageList
           sudo sealer build -f Kubefile -m lite -t kubernetes:$version-auto .
```
Comment (fanux) [inline .github/workflows/auto-build.yml:39]: $version-auto -> $version

**Review by fanux — APPROVED** [review]

**Pull request merged**